src/LibVLCSharp/Shared/Core/Core.Apple.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace LibVLCSharp.Shared
+{
+#if APPLE
```

Comment:
Why isn't it at the beginning of the file, as the other files?

Reply:
Because that branch originally included the macOS rework, and there were different code paths for iOS/tvOS and macOS.

Fixed, that will come later.